### PR TITLE
diskrsync: init at 20170927

### PR DESCRIPTION
**Pull request by jluttine**
###### Motivation for this change

**~~NOTE: Do not merge yet. This is work in progress, doesn't work yet. I hope to get some help and feedback here.~~** Fixed. Feel free to merge.

Diskrsync is rsync for block devices and disk images: https://github.com/dop251/diskrsync

~~I get the following error when trying to build:~~ Fixed. I had a typo.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by jluttine** [comment]
I fixed the issue I originally had. Was just a stupid minor typo.

File: pkgs/tools/backup/diskrsync/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ buildGoPackage, fetchFromGitHub, stdenv }:
+
+buildGoPackage rec {
+
+  name = "${pname}-${version}";
+  pname = "diskrsync";
+  version = "20170927";
```
Comment (nlewo) [inline pkgs/tools/backup/diskrsync/default.nix]: Could you please use the name format `NAME-unstable-YYYY-MM-DD` as specified in https://nixos.org/nixpkgs/manual/#sec-package-naming

**Review by nlewo — CHANGES_REQUESTED** [review]

**Comment by jluttine** [comment]
Could this be merged or is there something I'd need to fix?

File: pkgs/tools/backup/diskrsync/default.nix (hunk 1 above)
Comment (jluttine) [reply pkgs/tools/backup/diskrsync/default.nix]: Fixed.

**Review by nlewo — APPROVED** [review]

**Comment by jluttine** [comment]
Updated the package.

**Comment by grahamc** [comment]
@GrahamcOfBorg build diskrsync

**Pull request merged**